### DISCUSSION TITLE
OpenGL Cores

### DISCUSSION THREAD

**Discussion by rtomasa** [libretro/ludo#500] [General]
Hi @kivutar , I'm also developing a minimal libretro frontend targeted only to Raspberry. I can start glcores already but I'm facing a general performance issue in N64, DC and even PSX. Not comply sure if it is due to something related to my implementation of OpenGLES or any of the retro environment commands that is not properly implemented or missing. I'd like to ask you if you found any performance issue when testing internally the GL cores too (I know the changes are not merged yet).

**Comment by kivutar** [comment]
I'm discovering this message 2 years late, sorry for the late reply.
I haven't noticed any performance issues, however, I was not testing on a machine as weak as a RPi.
I would need to see your code to help.
